src/handlers/dhcp/DhcpStreamHandler.h

```diff
@@ -46,7 +48,7 @@ class DhcpMetricsBucket final : public visor::AbstractMetricsBucket
     DhcpMetricsBucket()
     {
         set_event_rate_info("dhcp", {"rates", "total"}, "Rate of all DHCP wire packets (combined ingress and egress) per second");
```

Comment:
@rboucher-ns1  should we rename rates metrics as well?

Reply:
Why do you think they might need to be renamed?

Reply:
same reason as bellow, I will rename to rates and add throughput as well